panda/board/drivers/can.h

```diff
@@ -86,15 +86,15 @@ int can_err_cnt = 0;
   uint8_t can_num_lookup[] = {0,1,2,-1};
   int8_t can_forwarding[] = {-1,-1,-1,-1};
   uint32_t can_speed[] = {5000, 5000, 5000, 333};
-  bool can_autobaud_enabled[] = {false, false, false, false};
```

Comment:
@energee , I can remove those changes, right?
we don't need autobaud, do we?

Reply:
Yes, this runs perfectly on RELEASE panda code.